gaeguli/pipeline.c

```diff
@@ -1049,7 +1149,10 @@ gaeguli_pipeline_remove_target (GaeguliPipeline * self, GaeguliTarget * target,
     goto out;
   }
 
-  gaeguli_target_unlink (target);
+  gst_device_monitor_stop (self->device_monitor);
+
+  gaeguli_target_stop (target);
+
   if (gaeguli_target_get_state (target) == GAEGULI_TARGET_STATE_STOPPING) {
```

Comment:
I believe, this is not required now as gaeguli_target_stop () sets NULL state on the target pipeline and sets the STOPPED state.

Reply:
Sure, target pipeline is now separated and we can simply stop it without affecting the video source.